src/test/gdb_qpasssignals.py

```diff
@@ -0,0 +1,26 @@
+from turtle import st
```

Comment:
What's this for? It breaks CI (Tkinter not installed) and doesn't seem to be used.

Reply:
Apologies, that seems to have been auto imported by my IDE when I made a typo